### DISCUSSION TITLE
Installation guide complex

### DISCUSSION THREAD

**Discussion by JHarrison712** [nautobot/nautobot#5633] [General]
I'm trying out Nautobot coming from Netbox and it appears that the installation guide is not as simple as Netbox makes it. Is there a dummier guide out here? Also I noticed the slack is no longer available to get in touch is that correct? https://i.imgur.com/HvmQ1uw.png

**Comment by jvanderaa** [comment]
@JHarrison712 the slack is still present - https://slack.networktocode.com/

**Reply by jvanderaa** [reply]
Also just networktocode.slack.com if already registered.
